test/lib/ufe/testGroupCmd.py

```diff
@@ -241,6 +241,75 @@ def testGroupKind(self):
         self.assertEqual(Usd.ModelAPI(newGroupPrim).GetKind(), "")
         self.assertFalse(newGroupPrim.IsModel())
 
+    @unittest.skipUnless(mayaUtils.mayaMajorVersion() >= 2022, 'Grouping restriction is only available in Maya 2022 or greater.')
```

Comment:
Does this really on Maya version, or is does it need UFE v3?

Reply:
There is no UFE v3 required for changes in this PR.   The Maya side of changes which captures this exception is now in both master and release/2022 branch.

Reply:
Also as a note, merge order doesn't really matter in the case of this PR.